<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDishWindow</class>
 <widget class="QMainWindow" name="AddDishWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dishAddButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ê∑ªÂä†</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dishDeleteButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Âà†Èô§</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ÂïÜÈì∫ËèúÂìÅüå≠</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="dishTableWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>681</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dishRefreshButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>73</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Âà∑Êñ∞</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
